packages/dd-trace/src/opentracing/propagation/text_map.js

```diff
@@ -91,7 +91,9 @@ class TextMapPropagator {
     const header = tags.join(',')
 
     if (header.length <= 512) {
-      carrier[tagsKey] = header
+      if (header.length >= 1) {
```

Comment:
This could be in a single condition with the above, for example `if (header.length >= 1 && header.length <= 512)`. It should also have a corresponding test to avoid future regressions.